src/formattedcode/format_spdx_tv.py

```diff
@@ -0,0 +1,188 @@
+#
```

Comment:
Same as for JSON. IMHO duplicating the code in two modules is not worth it and error prone.
There should be two hook implementations within one module rather and shared code would not be duplicated then.